public/pay/bundled-card-validator.js

```diff
@@ -191,6 +191,15 @@ navigator.serviceWorker.addEventListener('message', e => {
 });
 navigator.serviceWorker.controller.postMessage('payment_app_window_ready');
 
+function pingServiceWorkerAfterDelay() {
+  window.setTimeout(() => {
+    navigator.serviceWorker.controller.postMessage('ping');
+    pingServiceWorkerAfterDelay();
+  }, 60000);  // 1 minute.
+}
+
+pingServiceWorkerAfterDelay();
```

Comment:
I wonder if a recursive call of the same function will cause the momery piled up. How about using `setInterval` instead?

```suggestion
const ping = function pingServiceWorkerAfterDelay() {
  navigator.serviceWorker.controller.postMessage('ping');
}

setInterval(pingServiceWorkerAfterDelay, 60000);
```

Reply:
Great idea! Thank you 👍